sdk/core/azure-core/src/main/java/com/azure/core/annotation/Generated.java

```diff
@@ -19,7 +19,9 @@
  * <p>This annotation is expected to be used in classes that are annotated with {@link ServiceClient} only.</p>
  */
 @Retention(SOURCE)
-@Target({METHOD, CONSTRUCTOR, FIELD})
+@Target({
+    METHOD, CONSTRUCTOR, FIELD
+})
```

Comment:
I don't think this is necessary - I'm a fan of single-line annotations whenever necessary.

Reply:
Agreed, I'll look into getting this changed. Want to dig into this further, say we get a case in test code with

`@ValueSource(strings = { "string", "anotherString", "somethingReallyLongThatGoesPastLineBreakLength"})` how would you want to see that formatted?

```java
// Option 1
@ValueSource(
    strings = {/* assume it fits now */}
)

// Option 2
@ValueSource(strings = { /* some strings */,
    /* more strings */})

// Option 3
@ValueSource(strings = {
    // String 1,
   // String 2,
   // etc
})

// Something else?
```

Reply:
I would only wrap when we are forced to, and then in that case it would be a new line per value, like we do with method definitions:

```
@ValueSource(foo = "foo", strings = {
    "String 1",
    "String 2",
    "AnExtremelyLongLineThatWillPushPastTheLineCharacterLimit"
}, bar = "bar", baz = "baz")
```

Otherwise, I would always prefer to see this:

```
@ValueSource(foo = "foo", strings = { "String 1", "String 2"}, bar = "bar", baz = "baz")
```